src/PRBProxy.sol

```diff
@@ -89,14 +97,27 @@ contract PRBProxy is
             revert PRBProxy_ExecutionUnauthorized({ owner: owner, caller: msg.sender, target: target });
         }
 
-        // Check that the target is a contract.
+        // Delegate call to the target contract, and handle the response.
+        response = _execute(target, data);
+    }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                          INTERNAL NON-CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @notice Executes a DELEGATECALL to the provided `target` with the provided `data`.
+    /// @dev Shared logic between the constructor and the {execute} function.
```

Comment:
Aren't the braces used when {contract.foo}?

Reply:
Good catch - I will change this to:

````
`execute`
````